package.json

```diff
@@ -31,5 +28,11 @@
   },
   "dependencies": {
     "handlebars": "^4.0.6"
+  },
+  "externals": {
```

Comment:
shouldn't this be an array? In order to enable oc.require to do things in the right order, example React and then ReactDOM?